e2e_tests/tests/fixtures/pytorch-rng-saver/model_def.py

```diff
@@ -0,0 +1,65 @@
+import random
+from typing import Any, Dict, Sequence, Tuple, Union
+
+import numpy as np
+import torch
+from torch import nn
+
+from determined.pytorch import DataLoader, PyTorchTrial
+
+TorchData = Union[Dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor]
+
+LR_START = 0.1
+INPUT = 1
+
+
+class IndexDataset(torch.utils.data.Dataset):
+    def __len__(self) -> int:
+        return 64
+
+    def __getitem__(self, index: int) -> Tuple:
+        return torch.Tensor([float(INPUT)])
+
+
+def yellow(msg):
+    return "\x1b[33m" + msg + "\x1b[m"
+
+
+class NoopPytorchTrial(PyTorchTrial):
+    def __init__(self, context):
+        self.context = context
+
+        model = nn.Linear(1, 1, False)
+        model.weight.data.fill_(0)
+
+        self.model = context.wrap_model(model)
+
+        opt = torch.optim.SGD(self.model.parameters(), LR_START)
+        self.opt = context.wrap_optimizer(opt)
+
+    def train_batch(
+        self, batch: TorchData, epoch_idx: int, batch_idx: int
+    ) -> Dict[str, torch.Tensor]:
+        w_real = self.model.weight.data[0]
+
+        loss = torch.nn.L1Loss()(self.model(batch), batch)
```

Comment:
nit: use `torch.nn.MSELoss()`, since it's easier to hand-calculate (and that's what I just used when I introduced this model as a unit test)